docs/sql-ref-syntax-qry-select-limit.md

```diff
@@ -91,7 +91,21 @@ SELECT name, age FROM person ORDER BY name LIMIT length('SPARK');
 
 -- A non-foldable expression as an input to LIMIT is not allowed.
 SELECT name, age FROM person ORDER BY name LIMIT length(name);
-org.apache.spark.sql.AnalysisException: The limit expression must evaluate to a constant value ...
+org.apache.spark.sql.AnalysisException
+{
+  "errorClass" : "_LEGACY_ERROR_TEMP_2400",
```

Comment:
since we are documenting this error, can we assign an official error class name to it?

Reply:
BTW, is this really what the end users will see?